### ISSUE TITLE
Customise the order of codeActionsOnSave inc. formatOnSave

### ISSUE THREAD

**Issue by benchesh** [microsoft/vscode#105301]
I've become used to using the default `vscode.typescript-language-features` as my JS formatter. However, the organisation I work for use eslint as a formatter. Ideally I'd like to not override my global formatter.

The `dbaeumer.vscode-eslint` extension offers a fantastic solution, _on paper_:

```
"editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
}
```

This will effectively resolve any formatting conflicts with the built-in formatter by fixing the linting errors when I save in projects that use eslint. Problem is, I also like to use `editor.formatOnSave`. I believe this actually runs _after_ `editor.codeActionsOnSave` which results in eslint's tweaks being instantly reverted. The order `editor.codeActionsOnSave` executes can indeed be customised by setting it as an array instead of as an object, however `editor.formatOnSave` is its own bespoke setting, and seems to always execute after `editor.codeActionsOnSave`.

To summarise, I want to be able to set `editor.codeActionsOnSave` to execute before `editor.formatOnSave`. Potential solutions could be:
- A setting like `editor.formatOnSaveAfterCodeActionsOnSave`, true by default
- The ability to add `editor.formatOnSave` to the `codeActionsOnSave` array

Thanks :)

**Comment by mjbvz** [comment]
Falls under #67147. Will follow up on that issue
